ui/app/rif/pages/lumino/luminoNetworkDetails/index.js

```diff
@@ -0,0 +1,153 @@
+import React, {Component} from 'react'
+import PropTypes from 'prop-types';
+import {connect} from 'react-redux'
+import rifActions from '../../../actions';
+import LuminoChannelItem from '../../../components/luminoChannelItem';
+import OpenChannel from '../../../components/lumino/open-channel';
+import {GenericTable} from '../../../components';
+
+const styles = {
+  myLuminoChannels: {
+    title: 'n-table-title',
+    table: 'n-table',
+    tbodyTd: 'n-table-td',
+    pagination: {
+      body: 'n-table-pagination',
+      buttonBack: 'n-table-pagination-back',
+      indexes: '',
+      activePageButton: 'n-table-pagination-active',
+      inactivePageButton: 'n-table-pagination-inactive',
+      buttonNext: 'n-table-pagination-next',
+    },
+  }
+}
+
+class LuminoNetworkDetails extends Component {
+
+  static propTypes = {
+    networkSymbol: PropTypes.string,
+    tokenNetwork: PropTypes.string,
+    getUserChannels: PropTypes.func,
+    getNetworkData: PropTypes.func,
+    tokenAddress: PropTypes.string,
+    networkName: PropTypes.string,
+  }
+
+  constructor (props) {
+    super(props);
+    this.state = {
+      loading: true,
+      networkData: {
+        channels: 0,
+        nodes: 0,
+      },
+      userChannels: [],
+    }
+  }
+
+  reloadChannelsData = async () => {
+    const {getUserChannels, tokenAddress, getNetworkData} = this.props;
+    const userChannels = await getUserChannels(tokenAddress);
+    if (userChannels && userChannels.length) this.setState({userChannels, loading: false})
+    const networkData = await getNetworkData(tokenAddress);
+    if (networkData) this.setState({networkData})
+  }
+
+
+  async componentDidMount () {
+    await this.reloadChannelsData();
+    // TODO: Run again a function to resolve addresses to RNS domains
```

Comment:
Is this Todo already fixed?

Reply:
No, we should open a ticket because it is a completly different issue.
As well we have to weight its impact on the sprint